JSQMessagesViewController/Categories/UIViewController+JSQMessages.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Created by Jesse Squires
+//  http://www.jessesquires.com
+//
+//
+//  Documentation
+//  http://cocoadocs.org/docsets/JSQMessagesViewController
+//
+//
+//  GitHub
+//  https://github.com/jessesquires/JSQMessagesViewController
+//
+//
+//  License
+//  Copyright (c) 2014 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+#import <UIKit/UIKit.h>
+
+@interface UIViewController (JSQMessages)
```

Comment:
I can definitely make it private. It's just an Apple bug not a JSQ bug, so this is a workaround. We can make it opt-in for sure.